azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java

```diff
@@ -1954,6 +1955,10 @@ private void ajaxHandleUpload(final HttpServletRequest req, final HttpServletRes
       });
 
       registerErrorsAndWarningsFromValidationReport(resp, ret, reports);
+      // Reload the flow_filter to ensure if this project is added to the file, then the filter is current.
+      ContainerizedDispatchManager containerizedDispatchManager =
```

Comment:
Add a if before this reload.
`if (this.executorManagerAdapter instanceof ContainerizedDispatchManager) {`